### ISSUE TITLE
Flann package downloaded from artifactory contains big unused files

### ISSUE THREAD

**Issue by JulianThijssen** [ManiVaultStudio/t-SNE-Analysis#38]
We use flann.lib, but the packages downloaded from artifactory also contain flann_s.lib which are very big files but are not used.

**Comment by alxvth** [comment]
More of a workaround: When building HDILib locally, nothing is downloaded from the artifactory, see #99.

In the meantime, a newer flann version is downloaded from the artifactory by default (1.9.2 instead of 1.8.5, when this issue was opened). Is this still an issue?

**Comment by JulianThijssen** [comment]
It seems to still be an issue. There is still a big flann_s.lib in my build folders.

**Comment by bldrvnlw** [comment]
You are correct it is large the whole flann package on windows compressed - (it includes Release/Debug and RelWithDebInfo) it is 34MB approx. The s.lib and (in Debug) the pdbs are surprisingly large (probably template bloat). However is it really causing an issue or does it just not look nice?
